docs/src/man/gen_prog_align.md

```diff
@@ -0,0 +1,22 @@
+# GenProgAlign
```

Comment:
I'd add more explanation here, it's hard to get what the code is doing in a short amount of time. You could add a pseudocode description for the algorithm.